integration-cli/docker_cli_set_unix_test.go

```diff
@@ -0,0 +1,121 @@
+// +build !windows
```

Comment:
same here - I think we want this to run on windows CLI too - just with a linux server.